IO/Ingestor/JsonIngestor.cs

```diff
@@ -0,0 +1,73 @@
+﻿using CommunAxiom.Commons.Ingestion.Attributes;
+using CommunAxiom.Commons.Ingestion.Configuration;
+using CommunAxiom.Commons.Ingestion.Validators;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace CommunAxiom.Commons.Ingestion.Ingestor
+{
+    [IngestionType(IngestorType.JSON)]
+    public class JsonIngestor : IngestorBase, IIngestor
+    {
+        private IEnumerable<DataSourceConfiguration> _configurations;
```

Comment:
Shouldn't this should be IEnumerable<FieldMetadata> as this is what holds the dataset's field configurations?